Remotion/Web/Core/Res/HTML/ExecutionEngine.js

```diff
@@ -64,6 +64,14 @@ function WxePage_Context(
   {
     ArgumentUtility.CheckNotNullAndTypeIsBoolean('hasSubmitted', hasSubmitted);
     ArgumentUtility.CheckNotNullAndTypeIsBoolean('isCached', isCached);
+
+    let dmaWxePostBackSequenceNumberField = document.getElementById('dmaWxePostBackSequenceNumberField');
+
+    if (dmaWxePostBackSequenceNumberField != null)
+    {
+      let postBackSequenceNumber = document.getElementById('wxePostBackSequenceNumberField').getAttribute('value');

```

Comment:
Ah, I just realized how we do it in SmartPage:

* access to value is done directly: document.getElementById('wxePostBackSequenceNumberField').value instead of get/setAttribute.

* In SmartPage.js / SmartPageInfo (ca line 330), we pass the ID of the hidden field into the javascript's ctor. And if the field does not exist (i.e. the dma field during 'normal' operations), the ID itself is already set to null.